demos/object_detection_demo_ssd_async/main.cpp

```diff
@@ -1,41 +1,126 @@
+static const char iou_thresh_output_message[] = "Optional. Filtering intersection over union threshold for overlapping boxes (YOLOv3 only).";
+static const char mt_message[] = "Model type: ssd or yolo";
+
+DEFINE_bool(h, false, help_message);
+DEFINE_string(i, "", video_message);
+DEFINE_string(m, "", model_message);
+DEFINE_string(d, "CPU", target_device_message);
+DEFINE_string(labels, "", labels_message);
+DEFINE_bool(pc, false, performance_counter_message);
+DEFINE_string(c, "", custom_cldnn_message);
+DEFINE_string(l, "", custom_cpu_library_message);
+DEFINE_bool(r, false, raw_output_message);
+DEFINE_double(t, 0.5, thresh_output_message);
+DEFINE_double(iou_t, 0.4, iou_thresh_output_message);
+DEFINE_bool(auto_resize, false, input_resizable_message);
+DEFINE_uint32(nireq, 2, num_inf_req_message);
+DEFINE_uint32(nthreads, 0, num_threads_message);
+DEFINE_string(nstreams, "", num_streams_message);
+DEFINE_bool(loop, false, loop_message);
+DEFINE_bool(no_show, false, no_show_processed_video);
+DEFINE_string(u, "", utilization_monitors_message);
+DEFINE_string(mt, "", mt_message);
+
+/**
+* \brief This function shows a help message
+*/
+static void showUsage() {
+    std::cout << std::endl;
+    std::cout << "object_detection_demo_ssd_async [OPTION]" << std::endl;
+    std::cout << "Options:" << std::endl;
+    std::cout << std::endl;
+    std::cout << "    -h                        " << help_message << std::endl;
+    std::cout << "    -i \"<path>\"               " << video_message << std::endl;
+    std::cout << "    -m \"<path>\"               " << model_message << std::endl;
+    std::cout << "      -l \"<absolute_path>\"    " << custom_cpu_library_message << std::endl;
+    std::cout << "          Or" << std::endl;
+    std::cout << "      -c \"<absolute_path>\"    " << custom_cldnn_message << std::endl;
+    std::cout << "    -d \"<device>\"             " << target_device_message << std::endl;
+    std::cout << "    -_fil \"<path>\"          " << labels_message << std::endl;
+    std::cout << "    -pc                       " << performance_counter_message << std::endl;
+    std::cout << "    -r                        " << raw_output_message << std::endl;
+    std::cout << "    -t                        " << thresh_output_message << std::endl;
+    std::cout << "    -auto_resize              " << input_resizable_message << std::endl;
+    std::cout << "    -nireq \"<integer>\"        " << num_inf_req_message << std::endl;
+    std::cout << "    -nthreads \"<integer>\"     " << num_threads_message << std::endl;
+    std::cout << "    -nstreams                 " << num_streams_message << std::endl;
+    std::cout << "    -loop                     " << loop_message << std::endl;
+    std::cout << "    -no_show                  " << no_show_processed_video << std::endl;
+    std::cout << "    -u                        " << utilization_monitors_message << std::endl;
+}
 
-using namespace InferenceEngine;
 
```

Comment:
```suggestion
```

Reply:
not clear

Reply:
Remove extra line.

Reply:
I think I've already removed it as I cannot see it in the code